libraries/AP_AHRS/AP_AHRS_Logging.cpp

```diff
@@ -144,23 +144,24 @@ void AP_AHRS_View::Write_AttitudeView(const Vector3f &targets) const
 }
 
 // Write a rate packet
-void AP_AHRS_View::Write_Rate(const AP_Motors &motors, const AC_AttitudeControl &attitude_control,
+void AP_AHRS_View::Write_Rate(const AP_Motors &motors, AC_AttitudeControl &attitude_control,
```

Comment:
Logging should be already fixed in a previous PR and moved to the Attitude Controller. WIP

Reply:
Fixed